tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es2017",
-    "lib": ["ES2018", "ES2019", "ES2020", "ESNext"],
+    "module": "CommonJS",
+    "target": "ES2021",
```

Comment:
In `scripts/build.ts`, you use `ES2022` L214 and on L240, but here you use `ES2021`. Don’t you want to use the same target? :thinking:

Reply:
Fixed 👍